frontend/src/views/SQR/Form.vue

```diff
@@ -488,18 +488,15 @@ export default class Form extends Vue {
     this.debounceSubmitDraft.cancel();
   }
 
-  async submitForm(save = false) {
-    const sqrSubmissionBranch = save
+  async submitForm(submit = false) {
+    const sqrSubmissionBranch = submit
       ? SubmissionsBranch.COMPLETED
       : SubmissionsBranch.DRAFTS;
-    if (!save) {
-      this.draftStatus = FormState.SAVING;
+    if (!submit) {
+      this.formStateMessages[FormState.SAVING] = "Saving...";
```

Comment:
Avoid redefining messages. The message for `FormState.SAVING` is not changing. Rather, introduce another state `FormState.SUBMITTING`.